game/ui/tileview/battletileview.h

```diff
@@ -107,22 +108,26 @@ class BattleTileView : public TileView
 	sp<Image> pathPreviewUnreachable;
 	std::list<Vec3<int>> pathPreview;
 	int pathPreviewTicksAccumulated = 0;
-	// -3 = unreachable
-	// -2 = too far
-	// -1 = no previewed path stored
-	// 0+ = path cost
-	int previewedPathCost = -1;
+	enum class PreviewedPathCostSpecial : int
```

Comment:
You're right, the lots of static_casts<> here are pretty ugly.

I'm then leaning towards something like:
constexpr int PREVIEW_PATH_UNREACHABLE = -1;
etc.

Thoughts?

Reply:
..but this is getting really nit-picky. The core idea (No magic numbers) has been solved. We can always change it later anyway